package Exception.lesson03;

public class Test {
    public static void main(String[] args) {
        try{

        }catch (Exception e){

        }
    }
}
